pom.xml

```diff
@@ -214,6 +214,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-resources-plugin</artifactId>
+                <version>3.3.0</version>
+                <configuration>
+                    <propertiesEncoding>ISO-8859-1</propertiesEncoding>
+                </configuration>
+            </plugin>
```

Comment:
This should be removed, it is already defined in pom.xml

Reply:
Just merge dev-2.x and it will probably go away.